mne/preprocessing/maxwell.py

```diff
@@ -41,6 +41,88 @@
 # differences between algorithms
 
 
+@verbose
+def maxwell_filter_prepare_emptyroom(
+    raw_er, *, raw, bads='from_raw', verbose=None
+):
+    """Prepare an empty-room recording for Maxwell filtering.
+
+    Empty-room data by default lacks certain properties that are required to
+    ensure running :func:`~mne.preprocessing.maxwell_filter` will process the
+    empty-room recording the same way as the experimental data. This function
+    preconditions empty-room raw data instance accordingly so it can be used
+    for Maxwell filtering.
+
+    Specifically, this function will:
+
+    * Compile the list of bad channels according to the ``bads`` parameter.
+    * Inject the device-to-head transformation matrix from the experimental
+      recording into the empty-room recording.
+    * Set the same montage on the empty-room recording as the experimental
+      recording.
```

Comment:
This is too long for the start of a function. I'd move all of this to the Notes section.

Reply:
@hoechenberger add a line to refer to Notes and move the "Specifically ..." to a Notes section at the botton.